desktop/core/src/desktop/lib/rest/raz_http_client_test.py

```diff
@@ -55,3 +55,21 @@ def test_get_file(self):
             clear_cookies=False,
             timeout=120
         )
+
+        # When there is no SAS token in response
+        raz_get_url.return_value = {}
+        client = RazHttpClient(username='test', base_url='https://gethue.blob.core.windows.net')
+        f = client.execute(http_method='GET', path='/gethue/data/customer.csv', params={'action': 'getStatus'})
+
+        raz_http_execute.assert_called_with(
```

Comment:
One feeling is that it is probably better to raise a clean 503 exception, as if there is no token, for sure it is going to fail?